dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrDataModel.ts

```diff
@@ -832,7 +832,7 @@ export const sfdrDataModel = [
             name: "femaleBoardMembers",
             label: "Female Board Members",
             description:
-              "Number of females on the board, i.e. means the administrative, management or supervisory body of a company.",
+              "Number of females on the board, i.e. means the administrative, management or supervisory body of a company",
```

Comment:
The DD seems to be rather inconsistent in punctuations

Reply:
Indeed, I don't like that either. However, I don't think that adding it again will improve anything in terms of consistency

Reply:
Yes it seems to be inconsistent throughout the DD. let's leave this issue out for now